package.json

```diff
@@ -35,7 +35,7 @@
     "dependencies": {
         "cypress": "^9.0.0",
         "cypress-file-upload": "^5.0.8",
-        "cypress-plugin-snapshots": "^1.4.4",
+        "cypress-visual-regression": "^3.0.0",
```

Comment:
Does changing the library change the syntax used in tests and break existing tests? If so, this should be released as a breaking change (the PR must contain a commit message with `breaking:` or `feat!:` prefix).

Otherwise I don't have any issue with the PR (haven't successfully run https://github.com/oat-sa/tao-deliver-fe/pull/868 yet though).

Reply:
In fact, it looks like this repo doesn't even have the conventional commits checker or automatic release actions.
So assuming you will release this manually, @marpesia, just make sure you include the bump to 2.0.0 within that process.